docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
It is a copy-paste version of that, except mentioning the use of online installer instead of offline one for the EMR cluster. After a brief discussion, we decided that it's better to duplicate the PA installation page, rather than just describing what is different for the EMR cluster, which may create confusion.